provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/endpoint/EndpointController.java

```diff
@@ -37,12 +37,11 @@
  */
 @RestController
 @RequestMapping(
-  value = {"/Endpoint"},
+  value = {"/stu3/Endpoint"},
```

Comment:
@jhulbert-va I added `/stu3/` to the path here and on `MetadataController`. This may affect the changes you're making to the provider-directory ingress routes